development/docker-compose/docker-compose.yaml

```diff
@@ -251,6 +254,7 @@ services:
       - bootstrap
     environment:
       <<: *common-variables
+      eureka.client.register-with-eureka: false
```

Comment:
with this, the bootstrap task can still use service discovery, but it will avoid registering itself to be discoverable, which makes sense, as it is not a service. It will avoid the errors in the logs when the task is shutting down.